test/fuzz/sql_fuzzer/sql_query_proto_to_string.cc

```diff
@@ -0,0 +1,965 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#include "sql_query_proto_to_string.h"
+
+#include <algorithm>
+#include <cstring>
+
+using namespace sql_query;
+
+/* Every conversation function from SQL Query type to std::string must end with
+ * a space symbol. Thus, no reserved or user words will accidentally
+ * concatenate.
+ */
+#define PROTO_TOSTRING(TYPE, VAR_NAME) \
+	std::string TYPE##ToString(const TYPE & (VAR_NAME))
+
+namespace sql_fuzzer {
+
+namespace {
+constexpr uint32_t kMaxColumnNumber = 20;
+constexpr uint32_t kMaxTableNumber = 8;
+constexpr uint32_t kMaxColumnConstraintNumber = 10;
+[[maybe_unused]]
+constexpr uint32_t kMaxTableConstraintNumber = 10;
+[[maybe_unused]]
+constexpr uint32_t kMaxIndexNumber = 10;
+} /* namespace */
+
+PROTO_TOSTRING(Select, select);
+PROTO_TOSTRING(FunctionExpression, expression);
+PROTO_TOSTRING(Term, term);
+PROTO_TOSTRING(PredicateExpression, expression);
+PROTO_TOSTRING(JoinSource, source);
+
+PROTO_TOSTRING(TableName, table_name) {
+	return "table"
+		+ std::to_string(table_name.code() % kMaxTableNumber) + " ";
```

Comment:
We usually leave the `+` and other binary operators on the previous line.

Reply:
Fixed:

```diff
@@ -37,13 +38,13 @@ PROTO_TOSTRING(PredicateExpression, expression);
 PROTO_TOSTRING(JoinSource, source);
 
 PROTO_TOSTRING(TableName, table_name) {
-       return "table"
-               + std::to_string(table_name.code() % kMaxTableNumber) + " ";
+       return "table" +
+               std::to_string(table_name.code() % kMaxTableNumber) + " ";
 }
 
 PROTO_TOSTRING(ColumnName, column_name) {
-       return "column"
-               + std::to_string(column_name.code() % kMaxColumnNumber) + " ";
+       return "column" +
+               std::to_string(column_name.code() % kMaxColumnNumber) + " ";
 }
 
 PROTO_TOSTRING(VarChar, varchar) {
```